### PR TITLE
DOC-4381: Add defintions for platform groups

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

File: docs/Administration.rst [hunk 1]
```diff
@@ -2272,6 +2272,22 @@ By default, all existing groups will be displayed on the left panel. The
 right panel shows information of the selected group and of its members
 with the Add Member form.
 
+.. _ExistingGroups:
+
+**Existing Groups**
+
+By default, in eXo Platform there are 2 parent groups: Organzation and Platform
+**Organization** helps you to organize your company members: Employees, managers...
```
Comment (plamarque) [inline docs/Administration.rst]: deparments, teams or any way to represent your organization

Comment (plamarque) [reply docs/Administration.rst]: also indicate that these groups have no special effects in the product. they are just here as placeholder

File: docs/Administration.rst [hunk 2]
```diff
@@ -2272,6 +2272,22 @@ By default, all existing groups will be displayed on the left panel. The
...
+**Platform** helps to define roles in the platform, it contains four sub groups:
+-  Administration: contains users having administrative roles i.e. who are able to access 
+   the Administration menu |image|.
+-  Rewarding Administrators: contains users having the right to :ref:`manage rewards <ManagingRewards>`
+-  Guests: contains external users to the platform, they have access only to public content of the platform
+-  Users: contains all the platform users able to connect to the platform.
+
+.. note: Any registred user is autolatically added to the platform/users group to be able to connect.
```
Comment (plamarque) [inline docs/Administration.rst]: automatically

Comment (plamarque) [reply docs/Administration.rst]: didnt you forget web contributors?

**Review by plamarque — DISMISSED** [review]

File: docs/Administration.rst (hunk 1 above)
Comment (mzorai) [reply docs/Administration.rst]: OK, thanks for the tip

File: docs/Administration.rst (hunk 2 above)
Comment (mzorai) [reply docs/Administration.rst]: yes, it is coming with the next commit

File: docs/Administration.rst [hunk 3]
```diff
@@ -2391,14 +2413,38 @@ Managing memberships
...
 
-By default, 9 membership types are available in PRODUCT including
-Member, Author, Editor, Manager, Redactor, Validator, Webdesigner,
-Publisher and \*.
+By default, 9 membership types are available in eXo Platform including
+Member, Author, Editor, Manager, Redactor, Validator, Webdesigner, Publisher and \*.
 
 |image178|
 
+The membeship, such as `manager:/platform/users`, is defined by both membership type (manager, member, author...)
```
Comment (cmagral) [inline docs/Administration.rst]: Typo here.

Comment (mzorai) [reply docs/Administration.rst]: OK

File: docs/Administration.rst [hunk 4]
```diff
@@ -2391,14 +2413,38 @@ Managing memberships
...
+
 .. note:: The \* membership is identical to any other type, so choosing this type means adding all available membership types.
 
+**Membership usage in eXo Platform**
+
+In eXo Platform, the most used memships are `manager`, `member` and `*`.
+
+- The `manager` role to define users with managing privileges
+- The `member` role to define normal users in a group
+- `*` defines the whole roles
```
Comment (ahamdi) [inline docs/Administration.rst]:
```suggestion
- `*` all the roles combined, that means a user with * membership type will have the permissions assigned to all  membership types within this group
```

**Review by ahamdi — CHANGES_REQUESTED** [review]

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**